### PR TITLE
Add support to get connection data from Trigger Authorization in PostgreSQL Scaler

### PR DESCRIPTION
**Pull request by JorTurFer**
Signed-off-by: jorturfer <jorge_turrado@hotmail.es>

Add support to get connection data from Trigger Authorization in PostgreSQL Scaler
### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#developer-certificate-of-origin-signing-your-work))
- [x] Tests have been added
- [x] A PR is opened to update our Helm chart ([repo](https://github.com/kedacore/charts)) *(if applicable, ie. when deployment manifests are modified)*
- [x] A PR is opened to update the documentation on ([repo](https://github.com/kedacore/keda-docs)) *(if applicable)* https://github.com/kedacore/keda-docs/pull/535
- [x] Changelog has been updated

Fixes https://github.com/kedacore/keda/issues/2110

### CODE REVIEW

**Review by zroubalik — APPROVED** [review]

File: pkg/scalers/postgresql_scaler.go [hunk 1]
```diff
@@ -225,3 +224,17 @@ func (s *postgreSQLScaler) GetMetrics(ctx context.Context, metricName string, me
 
 	return append([]external_metrics.ExternalMetricValue{}, metric), nil
 }
+
+func getFromAuthOrMeta(config *ScalerConfig, field string) (string, error) {
```
Comment (zroubalik) [inline pkg/scalers/postgresql_scaler.go:228]: This function should be defined in some other place outside of this scaler and referenced then.

**Review by zroubalik — CHANGES_REQUESTED** [review]

File: pkg/scalers/postgresql_scaler.go (hunk 1 above)
Comment (JorTurFer) [reply pkg/scalers/postgresql_scaler.go:228]: You are right, where should I define it? Maybe in nex file? At scaler_handler level?

Comment (zroubalik) [reply pkg/scalers/postgresql_scaler.go:228]: 🤷‍♂️ in `scaler.go` or maybe create utils package in `pkg/scalers` ?

**Comment by JorTurFer** [comment]
I'm working on this, I will reopen it when is ready

File: pkg/scalers/scaler.go [hunk 2]
```diff
@@ -81,3 +82,18 @@ type ScalerConfig struct {
 	// PodIdentity
 	PodIdentity kedav1alpha1.PodIdentityProvider
 }
+
+// This method helps getting a field from Auth or Meta sections
```
Comment (zroubalik) [inline pkg/scalers/scaler.go:86]:
```suggestion
// GetFromAuthOrMeta helps getting a field from Auth or Meta sections
```

**Review by zroubalik — APPROVED** [review]

**Pull request merged**